README.md

```diff
@@ -16,36 +16,29 @@
 
 ## 📦 What’s in the box?
 
-- [File uploader solutions](./solutions/file-uploader/) - file uploader implementations for the most frequent use cases.
+- [File uploader solutions](components/file-uploader/) - file uploader implementations for the most frequent use cases.
 - [Blocks to build your own custom solutions](./blocks/) - build your own integration solutions with simple HTML and CSS.
-- [Adaptive image solution](./solutions/adaptive-image/): all you need to distribute your web images efficiently.
-- [Cloud image editing solution](./solutions/cloud-image-editor/): tune your pictures on the fly.
+- [Adaptive image solution](components/adaptive-image/): all you need to distribute your web images efficiently.
+- [Cloud image editing solution](components/cloud-image-editor/): tune your pictures on the fly.
 - Web components for media content representation: audio, video, image galleries, etc.
 - etc.
 
 ## ⚙️ Installation and start
 
-### NPM
-
-`npm i @uploadcare/blocks`
-
-### Contribution
-
-1. Clone repository: `git clone git@github.com:uploadcare/blocks.git ./`.
-2. Run dependencies setup: `npm run setup`.
-3. Start local dev-server: `npm run dev`.
+Installation process is very easy and takes minutes.
+See our [Installation guide](/get-started/installation/).
 
 ## Typescript support
 
-We use [JSDoc type annotations](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html) for TypeScript static analysis support during development.
+We use <a href="https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html" target="_blank">JSDoc type annotations</a> for TypeScript static analysis support during development.
```

Comment:
It's better to setup md engine to create `target="_blank"` for all the external links instead of using raw html links.